File: Swift/TimeandDate/TimeandDate/ViewController.m
//
//  ViewController.m
//  TimeandDate
//
//  Created by Arthur on 2015-10-12.
//  Copyright © 2015 Arthur. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    timer = [NSTimer scheduledTimerWithTimeInterval:1.0 target:self selector:@selector(updateTimer) userInfo:nil repeats:YES];
}

-(void) updateTimer{
    NSDateFormatter *formatter=[[NSDateFormatter alloc]init];
    [formatter setDateFormat:@"hh:mm:ss"];
    self.timeLabel.text=[formatter stringFromDate:[NSDate date]];
    
    NSDateFormatter *formatter2=[[NSDateFormatter alloc]init];
    [formatter2 setDateFormat:@"YYYY-MM-dd"];
    self.dateLabel.text=[formatter2 stringFromDate:[NSDate date]];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Animations/Animations/ViewController.swift
//
//  ViewController.swift
//  Animations
//
//  Created by Arthur on 2015-11-01.
//  Copyright © 2015 Arthur. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var ImageView: UIImageView!
    
    var timer = NSTimer()
    var Counter = 0
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

 
    @IBAction func PlayAnimation(sender: AnyObject) {
        
        timer = NSTimer.scheduledTimerWithTimeInterval(0.05, target: self, selector: Selector("PerformAnimatin"), userInfo: nil, repeats: true)
        
        
    }
    
    func PerformAnimatin(){
        
        Counter++
        
        if Counter == 16{
            Counter = 1
        }
        
        ImageView.image = UIImage(named: "win_\(Counter).png")
        
    }
}

